frontend/birth_ui.py

```diff
@@ -0,0 +1,70 @@
+import tkinter
+from tkinter import ttk
+from tkinter import messagebox
+from typing import Any
+from tkcalendar import DateEntry
+from datetime import datetime
+import pytz
+import uuid
+
+from database_accesor import Database
+
+_TABLE_NAME = "BirthDB"

```

Comment:
Is the table name better as a global variable or defined within the class? I defined mine within the class. We need to align to ensure consistency in the different ui files.